crates/bdk/README.md

```diff
@@ -72,7 +72,7 @@ fn main() {
     let db = ();
 
     let descriptor = "wpkh(tprv8ZgxMBicQKsPdy6LMhUtFHAgpocR8GC6QmwMSFpZs7h6Eziw3SpThFfczTDh5rW2krkqffa11UpX3XkeTTB2FvzZKWXqPY54Y6Rq4AQ5R8L/84'/0'/0'/0/*)";
-    let mut wallet = Wallet::new(descriptor, None, db, Network::Testnet).expect("should create");
+    let mut wallet = Wallet::new(descriptor, None, db, Network::Testnet, None).expect("should create");
```

Comment:
I'd say either make new functions to support this or create a `Params` type or a `WalletBuilder` or something like that.  It's hard to figure out what these `None`s mean just by looking at the signature.

Reply:
Let's do this in a separate PR, once #1101 is completed also.

Reply:
Do what? Please don't add another param to this function in this PR.